zhaquirks/inovelli/__init__.py

```diff
@@ -238,6 +238,156 @@ def handle_cluster_request(
+            0x0052: ("default_led5_strip_intensity_when_on", t.uint8_t, True),
+            0x0053: ("default_led5_strip_intensity_when_off", t.uint8_t, True),
+            0x0055: ("default_led6_strip_color_when_on", t.uint8_t, True),
+            0x0056: ("default_led6_strip_color_when_off", t.uint8_t, True),
+            0x0057: ("default_led6_strip_intensity_when_on", t.uint8_t, True),
+            0x0058: ("default_led6_strip_intensity_when_off", t.uint8_t, True),
+            0x005A: ("default_led7_strip_color_when_on", t.uint8_t, True),
+            0x005B: ("default_led7_strip_color_when_off", t.uint8_t, True),
+            0x005C: ("default_led7_strip_intensity_when_on", t.uint8_t, True),
+            0x005D: ("default_led7_strip_intensity_when_off", t.uint8_t, True),
+            0x005F: ("default_all_led_color_when_on", t.uint8_t, True),
+            0x0060: ("default_all_led_color_when_off", t.uint8_t, True),
+            0x0061: ("default_all_led_intensity_when_on", t.uint8_t, True),
+            0x0062: ("default_all_led_intensity_when_off", t.uint8_t, True),
+            0x0100: ("local_protection", t.Bool, True),
+            0x0101: ("remote_protection", t.Bool, True),
+            0x0102: ("dimmer_onoff_mode", t.Bool, True),
+            0x0103: ("only_one_led_mode", t.Bool, True),
+            0x0104: ("firmware_progress_led", t.Bool, True),
+        }
+    )
+
+    server_commands = {
+        0x00: foundation.ZCLCommandDef(
+            "button_event",
+            {"button_pressed": t.uint8_t, "press_type": t.uint8_t},
+            is_reply=False,
+            is_manufacturer_specific=True,
+        ),
+        0x01: foundation.ZCLCommandDef(
+            "led_effect",
+            {
+                "led_effect": t.uint8_t,
+                "led_color": t.uint8_t,
+                "led_level": t.uint8_t,
+                "led_duration": t.uint8_t,
+            },
+            is_reply=False,
+            is_manufacturer_specific=True,
+        ),  # LED Effect
+        0x02: foundation.ZCLCommandDef(
+            "reset_energy_meter",
+            {},
+            is_reply=False,
+            is_manufacturer_specific=True,
+        ),  # LED Effect
+        0x03: foundation.ZCLCommandDef(
+            "individual_led_effect",
+            {
+                "led_number": t.uint8_t,
+                "led_effect": t.uint8_t,
+                "led_color": t.uint8_t,
+                "led_level": t.uint8_t,
+                "led_duration": t.uint8_t,
+            },
+            is_reply=False,
+            is_manufacturer_specific=True,
+        ),  # individual LED Effect
+        0x24: foundation.ZCLCommandDef(
+            "led_effect_complete",
+            {
+                "notification_type": t.uint8_t,
+            },
+            is_reply=False,
+            is_manufacturer_specific=True,
+        ),
+    }
+
+    def handle_cluster_request(
```

Comment:
hrm if 99% of this is the same maybe just extend the other one and override the attributes

Reply:
Have to update const.py in HA as well, the cluster rename breaks that piece.